src/states/common/approved.rs

```diff
@@ -248,6 +248,12 @@ pub trait Approved: Relocated {
                 NetworkEvent::PurgeCandidate(old_public_id) => {
                     self.handle_purge_candidate_event(old_public_id)?
                 }
+                NetworkEvent::ParsecPrune => {
+                    info!(
```

Comment:
Cannot really test logs:-), as a step through can add a counter and a getter (did that for message Ack).
so you can write the test case that exercise it even if prunning does not actually happen on consensus,